assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,170 @@
+      isToggleOn: !state.isToggleOn
+    }));
+
+    this.state.isToggleOn === true ? $('.panel').removeClass('visibility') : $('.panel').addClass('visibility');
+  }
+
+  toAttribute (attributeName) {
+    const { setAttributes } = this.props;
+    return value => {
+      setAttributes({ [attributeName]: value });
+    };
+  }
+
+  handleErrors (errors) {
+    this.setState(errors);
+  }
+
+  // renders the settings
+  renderEdit () {
+    const { attributes } = this.props;
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+            <TextControl
+              label={__('Button Text', 'planet4-blocks-backend')}
+              placeholder={__('Override button text', 'planet4-blocks-backend')}
+              help={__('Leave the button text empty to hide the button on the front page.', 'planet4-blocks-backend')}
+              value={attributes.accordion_btn_text}
+              onChange={this.toAttribute('accordion_btn_text')}
+            />
+            <URLInput
+              label={__('Button Link', 'planet4-blocks-backend')}
+              value={attributes.accordion_btn_url}
+              onChange={this.toAttribute('accordion_btn_url')}
+            />
+            <CheckboxControl
+              label={__('Open in a new Tab', 'planet4-blocks-backend')}
+              help={__('Open button link in new tab', 'planet4-blocks-backend')}
+              value={attributes.button_link_new_tab}
+              checked={attributes.button_link_new_tab}
+              onChange={this.toAttribute('button_link_new_tab')}
+            />
+          </PanelBody>
+        </InspectorControls>
+      </Fragment>
+    );
+  }
+
+  // renders the editor view
+  renderView () {
+    const { attributes } = this.props;
+
+    return <Fragment>
+      <div className="block accordion-block my-3">
+        <Tooltip text={__('Leave empty the title and/or description of this block if you want to hide them on the front page.', 'planet4-blocks-backend')}>
+          <header>
+            <RichText
+              tagName="h2"
+              className="page-section-header mt-3"
+              placeholder={__('Enter title', 'planet4-blocks-backend')}
+              value={attributes.accordion_title}
+              onChange={this.toAttribute('accordion_title')}
+              keepPlaceholderOnFocus={true}
+              withoutInteractiveFormatting
+              characterLimit={60}
+              multiline="false"
+            />
+          </header>
+        </Tooltip>
+        <RichText
+          tagName="p"
+          className="page-section-description"
+          placeholder={__('Enter description', 'planet4-blocks-backend')}
+          value={attributes.accordion_description}
+          onChange={this.toAttribute('accordion_description')}
+          keepPlaceholderOnFocus={true}
+          withoutInteractiveFormatting
+          characterLimit={200}
+          multiline="true"
```

Comment:
I still can't enter multiline text for the description 🤔 I think you can just remove this line completely and it should work!

Reply:
Not sure if that's causing the issue, but with a boolean prop you can just do
```js
<RichText
// ...
  multline>

Reply:
Yeah that should also work 🙂 But I think it's the default value anyway so it can just be removed!